src/window/windows/castlewindow_winapi.inc

```diff
@@ -199,6 +199,60 @@ const
   VK_OEM_8 = $DF;
   VK_OEM_102 = $E2;
 
+function TCastleWindow.WMDropFiles(var DropH: HDROP) : LRESULT;
+  
+var
+  DroppedFileCount: Integer;  // number of files dropped
+  FileNameLength: Integer;    // length of a dropped file name
+  FileName: string;           // a dropped file name
+  I: Integer;                 // loops thru all dropped files
+  // DropPoint: TPoint;       // point where files dropped
+  FileNames: array of AnsiString;
+begin
+  inherited;
```

Comment:
Remove `inherited` from this -- there is no `WMDropFiles` implemented in ancestor (and this is not even virtual method), so `inherited` would do nothing (and could confuse devs looking at it :) ).